cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -1936,6 +1937,76 @@ pub(crate) fn emit(
             sink.put4(0x0);
         }
 
+        Inst::JmpCondOr {
+            cc1,
+            cc2,
+            taken,
+            not_taken,
+        } => {
+            // Emit:
+            //   jcc1 taken
+            //   jcc2 taken
+            //   jmp not_taken
+            //
+            // Note that we enroll both conditionals in the
+            // branch-chomping mechanism because MachBuffer
+            // simplification can continue upward as long as it keeps
+            // chomping branches. In the best case, if taken ==
+            // not_taken and that one block is the fallthrough block,
+            // all three branches can disappear.
```

Comment:
Mind writing a test for this case where all three branches can get chomped and replaced with a fallthrough?